<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HausdorffFrechetClass</class>
 <widget class="QMainWindow" name="HausdorffFrechetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HausdorffFrechet</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action1_1"/>
    <addaction name="action1_2"/>
    <addaction name="action1_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>线匹配</string>
    </property>
    <addaction name="action2_1"/>
    <addaction name="action2_2"/>
    <addaction name="action2_3"/>
    <addaction name="action2_4"/>
    <addaction name="action2_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>结果评价</string>
    </property>
    <addaction name="action3_1"/>
    <addaction name="action3_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>图形</string>
    </property>
    <addaction name="action4_1"/>
    <addaction name="action4_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action1_1">
   <property name="text">
    <string>读取文件</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>保存报告</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action2_1">
   <property name="text">
    <string>基于传统Hausdorff距离</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>基于离散Frechet距离</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>基于离散Frechet距离</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>基于改进的Hausdorff_SMHD距离</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>基于平均Frechet距离ToolStripMenuItem</string>
   </property>
  </action>
  <action name="action3_1">
   <property name="text">
    <string>读入参考数据</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>评价</string>
   </property>
  </action>
  <action name="action4_1">
   <property name="text">
    <string>绘制图形</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>保存图形</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>基于改进的Hausdorff(SMHD)距离</string>
   </property>
  </action>
  <action name="action2_4">
   <property name="text">
    <string>基于平均Frechet距离</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>一键生成全部</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HausdorffFrechet.qrc"/>
 </resources>
 <connections/>
</ui>
